packages/core/src/repository/cache.repository.ts

```diff
@@ -109,7 +109,11 @@ export class CacheRepository<T> implements GetRepository<T>, PutRepository<T>, D
         }
     }
 
+    /**
+     * @deprecated please use get with an array type instead
+     */
     public async getAll(query: Query, operation: Operation): Promise<T[]> {
+        console.warn('getAll is deprecated. Please use get instead');
```

Comment:
Nit: mostly so that you're aware, most of these classes have a `logger` available, we used that previously to notify the user about deprecations. **No need to change anything as this will be gone soon enough.** e.g.:

```
this.logger.warning('[DEPRECATION] `deleteAll` will be deprecated. Use `delete` instead.');
```

`deleteAll` removal commit: a7cb31ced5eb3d19db3dab64016b886b64f178b2.

Reply:
Most IDEs already give a visual feedback when the deprecated annotation is added. In any case, better over communicating.